### DISCUSSION TITLE
Could not resolve com.mapbox.maps:android:10.18.0.

### DISCUSSION THREAD

**Discussion by kamal-techchart** [rnmapbox/maps#3579] [Q&A]
I am using rnmapbox version ^10.1.28 , 
How can i find the solution for this as all the token related things are fine
* What went wrong:
Could not determine the dependencies of task ':rnmapbox_maps:compileDebugAidl'.
> Could not resolve all task dependencies for configuration ':rnmapbox_maps:debugCompileClasspath'.
   > Could not resolve com.mapbox.maps:android:10.18.0.
     Required by:
         project :rnmapbox_maps
      > Could not resolve com.mapbox.maps:android:10.18.0.
         > Could not get resource 'https://api.mapbox.com/downloads/v2/releases/maven/com/mapbox/maps/android/10.18.0/android-10.18.0.pom'.
            > Could not GET 'https://api.mapbox.com/downloads/v2/releases/maven/com/mapbox/maps/android/10.18.0/android-10.18.0.pom'. Received status code 401 from server: Unauthorized
   > Could not resolve com.mapbox.mapboxsdk:mapbox-sdk-turf:6.11.0.
     Required by:
         project :rnmapbox_maps
      > Could not resolve com.mapbox.mapboxsdk:mapbox-sdk-turf:6.11.0.
         > Could not get resource 'https://api.mapbox.com/downloads/v2/releases/maven/com/mapbox/mapboxsdk/mapbox-sdk-turf/6.11.0/mapbox-sdk-turf-6.11.0.pom'.
            > Could not GET 'https://api.mapbox.com/downloads/v2/releases/maven/com/mapbox/mapboxsdk/mapbox-sdk-turf/6.11.0/mapbox-sdk-turf-6.11.0.pom'. Received status code 401 from server: Unauthorized
            
   my build.gradle file consist
   
   

> buildscript {
    ext {
        buildToolsVersion = "34.0.0"
        minSdkVersion = 23
        compileSdkVersion = 34
        targetSdkVersion = 34
        ndkVersion = "26.1.10909125"
        kotlinVersion = "1.9.22"
        RNMapboxMapsImpl = 'mapbox'
    }
    repositories {
        google()
        mavenCentral()
    }
    dependencies {
        // new added
        classpath 'com.google.gms:google-services:4.3.15'
        classpath 'com.google.firebase:firebase-crashlytics-gradle:2.9.2'
        classpath("com.android.tools.build:gradle")
        classpath("com.facebook.react:react-native-gradle-plugin")
    }
   allprojects {
    repositories {
        // ...other repos
        maven {
            url 'https://api.mapbox.com/downloads/v2/releases/maven'
            authentication {
                basic(BasicAuthentication)
            }
            credentials {
                // Do not change the username below.
                // This should always be `mapbox` (not your username).
                username = 'mapbox'
                // Use the secret token you stored in gradle.properties as the password
                password = project.properties['sk-key'] ?: ""
            }
        }
        // ...even more repos?
    }
}
}

**Comment by mfazekas** [comment]
> 401 from server: Unauthorized means that some of your tokens are not correct. Or doesn't have download permission

See
https://rnmapbox.github.io/docs/install?rebuild=android&v11-insructions=android&configure-module=android
